Added Readme

AALUND13 opened the pull request:
- Added Readme
- Rewrite Setup Symlink file

AALUND13 committed:
Added Readme

- Added Readme
- Rewrite Setup Symlink file

Tess-y reviewed (changes requested):
Honestly, I disagree with the sim-link idea in the first place and would have rejected it if I had had the branch protection setup correctly before you had committed it to main. That being said. If you _really_ want to implement it that way, you must do so in a way that can function on all systems, not just windows. Otherwise I am considering reverting the change made to the csproj file.

Scyye commented:
I dont see a need for a symlink file.
Is that not just making it so ya dont have to change a single variable in the csproj files, which takes ~10 seconds?

Tess-y commented:
To my knowledge, that is the reason it is there. Which is why if it was a PR it would have been rejected.

AALUND13 commented:
Since I don't want to figure out how to recreate the same setup in a shell file, I may remove the need for it to be a symlink and rewrite the README, excluding any references to the symlink. The changes will be made within a few minutes to a few hours.

AALUND13 commented:
I can use Python to create a symlink that works on Windows, Linux/UNIX, and macOS. However, the end user must have Python installed on their system. What do you think about using python to create symlink?

Scyye commented:
I dont see a reason for it.
Its fixing a non-issue, and having a weird requirement of python in order to develop a ROUNDS mod, is quite unnecessary

Tess-y commented:
I agree, you are attempting to needlessly complicate what was as simple as changing a single variable in a single file.

Tess-y commented:
if you can come up with a system that allows any user to do this without the need of installing additional external tools, then by all means go for it. but barring that I will be rejecting this pull request, and reverting the change that you made to the project file.

AALUND13 commented:
I am against using Python cause you have to install a extra software just to run one file

Tess-y commented:
> I am against using Python cause you have to install a extra software just to run one file

you are quite literally the one who suggested using python the first place. what was the point of suggesting it if you are against it?

AALUND13 committed:
Remove Symlink

I just going to remove the need for a symlink and rewrite the readme cause to a complex of a non-issue just to solve one miner non-issue only save a few seconds at the end

AALUND13 commented:
I just removed the need for a symlink and rewritten the readme cause to a complex of a non-issue just to solve one miner non-issue only save a few seconds at the end

Tess-y commented on README.md:
```diff
@@ -1,28 +1,25 @@
+- Navigate to the `RoundsModdingTools` folder.
+- Open the file named `RoundsModdingTools.csproj`.
+- Locate the line resembling: `<SteamFolder>F:\SteamLibrary</SteamFolder>`.
+- Replace `F:\SteamLibrary` with the path to your Steam Library folder.
```
Change lines 11 and 12 to just say to set the SteamFolder variable, rather then explicitly referencing _my_ Steam Library.
then I'll accept the PR.

AALUND13 committed:
Update README.md

Change "How to Build the Project" to "How to Setup the Project" and remove the "Build the Project" step
